content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,619 @@
+      - [Multiple Handlers with a Matching Contact](#multiple-handlers-with-a-matching-contact)
+      - [Multiple Handlers with a Single Matching Contact](#multiple-handlers-with-a-single-matching-contact)
+      - [Multiple Handlers with a Single Non-Matching Contact](#multiple-handlers-with-a-single-non-matching-contact)
+      - [Multiple Handlers with Multiple Matching Contacts](#multiple-handlers-with-multiple-matching-contacts)
+      - [Multiple Handlers with Multiple Non-Matching Contacts](#multiple-handlers-with-multiple-non-matching-contacts)
+      - [Multiple Handlers with Some Matching Contacts](#multiple-handlers-with-some-matching-contacts)
+- [Wrapping Up](#wrapping-up)
+  - [Contact Routing and Sensu Event Pipeline](#contact-routing-and-sensu-event-pipeline)
+- [References](#references)
+- [Additional Resources](#additional-resources)
+
+# Prerequisites
+
+Before diving into this guide, we recommend having the following components ready:
+
+- A working Sensu Enterprise deployment
+- Two or more handlers configured
+
+If you've not already signed up for Sensu Enterprise, you can do so via [this link][1].
+
+In this guide we'll be using [Slack][2] and [Email][3] handlers for our demonstration.
+
+# Overview
+
+Every incident, outage or event has an ideal first responder. This can be either
+a team or individual with the knowledge to triage and address the issue. Sensu
+Enterprise contact routing makes it possible to assign checks to specific teams
+and/or individuals, reducing mean time to response and recovery (MTTR). Contact
+routing works with all of the Sensu Enterprise third-party notification and
+metric integrations.
+
+_NOTE: Sensu Enterprise supports contact routing for all integrations except
+EC2, Puppet, Chef, Flapjack, and Event Stream._
+
+In this guide we'll cover configuring and using Sensu Enterprise Contact Routing.
+
+## Contact Routing Basics
+
+In Sensu Enterprise Contact Routing, contacts are composed of a name and
+configuration overrides for one or more of Sensu Enterprise's built-in
+integrations or handler configurations. A contact in Sensu Enterprise is not too
+dissimilar from a contact on your phone, in which they have a name and one or
+more _identifiers_ for various communication **channels** (e.g. a **phone**
+_number_, **email** _address_, **Twitter** _username_, etc). As an example your contact may have the following **email** _address, **slack** _channel_, **pagerduty** _service\_key_, etc.
+
+### Contact Configuration
+
+Configuring a contact requires you to define the name of the contact and the
+services plus configuration override(s) for that contact.
+
+{{< highlight json >}}
+{
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.com"
+      },
+      "slack": {
+        "channel": "#support"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+In the example above we have a contact named **support**. For **support** we
```

Comment:
I'm finding the intermixed use of bold and italics hard to follow. Since these are referencing named parameters, perhaps it makes more sense to use `back-ticks for code fencing` ? I think this is also what the [formatting section of our docs style guide specifies](https://github.com/sensu/sensu-docs/wiki/Sensu-docs-style-guide#formatting).

Reply:
If we're in agreement, let's make sure to apply that change to all the places we're using bold/italics.